### DISCUSSION TITLE
How can I disable a native <select>?

### DISCUSSION THREAD

**Discussion by wonu** [ariakit/ariakit#4308] [Questions]
I'm using a `<Select>` component rendered as a native element.
I want to prevent this element from being submitted with the form, but the `disabled` prop isn't working as expected.

```ts
<Ariakit.Select name="fruit" disabled /> // `disabled` attribute is not being set on the <select> element
```

Can you explain why and suggest a solution?

**Comment by diegohaz** [comment]
Moved to a feature request #4310
